modules/balana-core/src/main/java/org/wso2/balana/xacml3/Attributes.java

```diff
@@ -210,9 +210,9 @@ public void encode(StringBuilder builder) {
         builder.append("<Attributes Category=\"").append(category.toString()).append("\">");
 
         for(Attribute attribute : attributes){
-            if(attribute.isIncludeInResult()){
-                attribute.encode(builder);
-            }
+            //if(attribute.isIncludeInResult()){
```

Comment:
Then that is not the right place to check. With the check attributes without includeInResult are not printed in RequestCtx.encode().

I'll have to search for the part where attributes are added to the result later since I don't have time for it right now. Do you know where without having to research?